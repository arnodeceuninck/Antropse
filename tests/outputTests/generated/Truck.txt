Roadnetwork 0

Baan: A12
	-> snelheidslimiet vanaf positie 0: 120
	-> lengte: 5000

Voertuig: VRACHTWAGEN (ANT-432)
	-> baan: A12
	-> positie: 20
	-> snelheid: 0

-----------------------------------------------------------
