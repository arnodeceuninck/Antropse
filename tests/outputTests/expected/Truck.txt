Roadnetwork 0

Baan: A12
	-> snelheidslimiet: 120
	-> lengte: 5000

Voertuig: VRACHTWAGEN (ANT-432)
	-> baan: A12
	-> positie: 20
	-> snelheid: 0

-----------------------------------------------------------
